### ISSUE TITLE
/user/me/token is listing all tokens, not just API tokens

### ISSUE THREAD

**Issue by sungo** [joyent/conch#738]
As the subject says, `/user/me/token` is currently listing both `login_jwt` and named API tokens.

Sorry to have missed it in the PR process for #727. My original intention for this endpoint was to display API tokens only, not `login_jwt`.

I would like `/user/me/token` to return a list of API tokens, not auth/jwt tokens.

**Comment by karenetheridge** [comment]
Which endpoints, if any other, should ignore the existence of login tokens?  Should an implicit `api_only=1` be added to all endpoints? Should this option be explicitly supported to allow the inclusion of login tokens when desired?  I suspect it would be more confusing if some endpoints operated on all tokens and others operated only on api tokens (consider, for example, the existing `user/*/revoke` endpoints which until now were intended to operate on login tokens as that's all that existed).

**Comment by karenetheridge** [comment]
I propose this patch, to be included in #737:

```
diff --git a/lib/Conch/Controller/User.pm b/lib/Conch/Controller/User.pm
index 6b88dafcb..e4168033b 100644
--- a/lib/Conch/Controller/User.pm
+++ b/lib/Conch/Controller/User.pm
@@ -462,12 +465,26 @@ sub deactivate ($c) {
 
 Get a list of unexpired user tokens for the user.
 
+By default this returns api tokens only, but this can be adjusted with query parameters:
+
+ * ?login=1   (default: 0)
+ * ?api=1     (default: 1)
+
 Response uses the UserTokens json schema.
 
 =cut
 
 sub get_tokens ($c) {
+    my $with_login = $c->req->query_params->param('login') // 0;
+    my $with_api = $c->req->query_params->param('api') // 1;
+
+    # logically this would yield a null result
+    return $c->status(404) if not $login and not $api;
+
     my $rs = $c->stash('target_user')->user_session_tokens->unexpired->order_by('name');
+    $rs = $rs->login_only if $login and not $api;
+    $rs = $rs->api_only if not $login and $api;
+
     return $c->status(200, [ $rs->all ]);
 }
 
```

**Comment by sungo** [comment]
I have no use cases for exposing login tokens to the user. 

We need the ability, in the context of security, to revoke login tokens. In my opinion, this is the only context in which it makes sense to expose login tokens.
